src/containers/proficiency-level-select/ProficiencyLevelSelect.tsx

```diff
@@ -0,0 +1,81 @@
+import {
+  Checkbox,
+  FormControl,
+  FormHelperText,
+  InputLabel,
+  ListItemText,
+  MenuItem,
+  OutlinedInput,
+  Select,
+  SelectProps,
+  SxProps
+} from '@mui/material'
```

Comment:
better use default imports instead of named:
https://mui.com/material-ui/guides/minimizing-bundle-size/

Reply:
Thanks for suggestion, makes sense to use `default` imports instead. Fixed